task_01.rst

```diff
@@ -0,0 +1,131 @@
+
+#############################################
+  Why I Love Python
+#############################################
+
+Why I love **Python**, a language worth knowing.
+
+
+
+
+##################
+Running It
+##################
+
+Python syntax is really quite simple to execute. Anyone can do it just by executing the
+``python`` command on the command line and invoking the interpreter.
+
+You can also execute python programs by calling the program filename such as:
+
+.. code:: console  
+
+   me@mymachine $ python my_file_name.py
+    
+
+
+##################
+The Syntax
+##################
+
+Python syntax is very easy to write. Take a look at this simple statement:
+
+.. code:: python
+
+   >>> print 'Hello!' 
+
+
+##################
+The People
+##################
+
+It's amazing how nice people are in the Python development world. Just as with any
+community, however, it's important to follow certain rules of decorum.
+
+- Be courteous to others
+
+- Always read the manual before asking questions
+
+- Be succinct in your questions
+
+  - Don't give unnecessary details if they aren't needed
+
+
+  - Don't start a question with an apology or an announcement that you're going to ask the question
+
+  *Just ask the question!*
```

Comment:
A little note for your bold section, the indentation of a continuation of the same paragraph should start where the characters start. Right now, yours lines up with the `-` but you'll really want it to line up with the `D`